src/docdb/tree/DocDBStoredProcedureTreeItem.ts

```diff
@@ -73,4 +74,17 @@ export class DocDBStoredProcedureTreeItem extends AzExtTreeItem implements IEdit
         const client = this.root.getCosmosClient();
         await this.parent.getContainerClient(client).scripts.storedProcedure(this.id).delete();
     }
+
+    public async execute(context: IActionContext, partitionKey: string, parameters?: any[]): Promise<void> {
+        const client = this.root.getCosmosClient();
+        const result = await this.parent.getContainerClient(client).scripts.storedProcedure(this.id).execute(partitionKey, parameters);
+
+        try {
+            const resultFileName = `${this.label}-result`;
+            const text = JSON.stringify(result, undefined, 2);
+            await vscodeUtil.showNewFile(text, resultFileName, ".json");
```

Comment:
I think it'd be better to use our `openReadOnlyJson` function here instead. Since it's readonly it doesn't show as unsaved like `showNewFile` does.

Reply:
Done. And I used a randomly generated hex string for the unique document id since the result is supposed to be used as a temporarily.